### PR TITLE
Brandon/blog style update

### PR DESCRIPTION
**Pull request by brandonhaslegs**
- Updated the styling of the blog listing page to include GIFs and different gradients per post.
- Removed the word "BLOG" from the top of the page
- added the meta info to the single blog post page

### CODE REVIEW

File: docs/privacy.html [hunk 1]
```diff
@@ -118,7 +118,7 @@ <h2>Github Pages:</h2>
...
+          <p>We use Ungleich to host our instance of Matrix and our Matrix homeserver. A username and display name is collected to give a user access to the chat room on our Matrix homeserver, then all contributions by the user to the Radicle chat room are stored on our homeserver. Optionally, a user can create a Matrix account on our homeserver, in which case Ungleich may store an email for the purpose of recovering their password as well as the user’s IP address. Our Matrix homeserver is hosted in Ungleich���s own datacenter in Switzerland. For further information about how Ungleich handles security and privacy concerns, visit <a href="https://redmine.ungleich.ch/projects/open-infrastructure/wiki/Security_and_Privacy_Policy" target="_blank">this site</a>.</p>
```
Comment (juliendonck) [inline docs/privacy.html:121]:
```suggestion
          <p>We use Ungleich to host our instance of Matrix and our Matrix homeserver. A username and display name is collected to give a user access to the chat room on our Matrix homeserver, then all contributions by the user to the Radicle chat room are stored on our homeserver. Optionally, a user can create a Matrix account on our homeserver, in which case Ungleich may store an email for the purpose of recovering their password as well as the user’s IP address. Our Matrix homeserver is hosted in Ungleich's own datacenter in Switzerland. For further information about how Ungleich handles security and privacy concerns, visit <a href="https://redmine.ungleich.ch/projects/open-infrastructure/wiki/Security_and_Privacy_Policy" target="_blank">this site</a>.</p>
```

**Comment by juliendonck** [comment]
Made some cleanups to the code. @cloudhead could you share your ffmpeg settings for the gifs so I can convert the new ones to also be videos.

**Comment by juliendonck** [comment]
@brandonhaslegs there are some mobile styles missing for the overview page. The gifs don't scale down and it creates the much dreaded horizontal scroll

**Review by juliendonck — DISMISSED** [review]

**Review by juliendonck — DISMISSED** [review]

**Comment by cloudhead** [comment]
* The title of the article listing is just "radicle"
* Some of the titles on that page are cropped, eg. the first one.
* I find the title sizes a too big, makes it hard to navigate that page -- I can't imagine there being more than 3 posts on there, as it'll require a lot of scrolling.

**Comment by juliendonck** [comment]
closes #129 
https://github.com/radicle-dev/radicle.xyz/pull/171/commits/688bccbf08b89dc8b729305c4ab04a0129b0b7b2 & https://github.com/radicle-dev/radicle.xyz/pull/171/commits/acc08bb4ed49ea59671198c3840a8dbc5a471135

**Comment by juliendonck** [comment]
https://github.com/radicle-dev/radicle.xyz/pull/171/commits/fefcce9ba1d66eceabddfa13810e1c24c07f4ec1 closes #173

**Comment by juliendonck** [comment]
https://github.com/radicle-dev/radicle.xyz/pull/171/commits/3ec8c34c0de649f542489d2d738c93b0aca28dfb closes #174

**Comment by juliendonck** [comment]
https://github.com/radicle-dev/radicle.xyz/pull/171/commits/8817f98c66a5d213f2fb9294027ec5adcda99dd7 closes #149

**Comment by juliendonck** [comment]
What's left: 
- [ ] blog overview text size
- [ ] blog overview blend mode

**Comment by abbey-titcomb** [comment]
Just checked this out locally. A couple comments:

- I think lists, like bullets & numbered lists, need an indentation

- Does anybody else think that the margins are too tight? I think we could widen them a bit - it seems really skinny to me right now

- Should we re-include the footer at the end of each blog post?

- The Druk text works, unless the title is too long, then it kind of stacks weird. Maybe we change the header text in the blog post to be something more normie? Or make it take up less lines?

Besides that, looks great. Lmk how I can help

**Comment by juliendonck** [comment]
https://github.com/radicle-dev/radicle.xyz/pull/171/commits/b684d030d4cbf47512fe7a481476fb221feef0c3 closes #181

**Review by juliendonck — APPROVED** [review]

**Pull request merged**